test/src/client_utils.dart

```diff
@@ -233,7 +233,11 @@ abstract class _Harness {
     toClient.close();
   }
 
-  void sendResponseHeader({List<Header> headers = const []}) {
+  void sendResponseHeader() {
+    final headers = [
+      Header.ascii(':status', '200'),
+      Header.ascii('content-type', 'application/grpc'),
+    ];
```

Comment:
I noticed that the function arguments were never used. But I’m also not sure if this is the right way to go.

Reply:
Yeah, I think this is good fix.